library/Notifications/Widget/ItemList/IncidentHistoryListItem.php

```diff
@@ -31,38 +28,34 @@ class IncidentHistoryListItem extends BaseListItem
     protected function assembleVisual(BaseHtmlElement $visual): void
     {
         $incidentIcon = $this->getIncidentEventIcon();
-        if ($this->item->type === 'incident_severity_changed' || $this->item->type === 'opened') {
-            $content = new Icon($incidentIcon, ['class' => 'severity-' . $this->item->new_severity]);
-        } elseif ($this->item->type === 'rule_matched') {
-            $content = new IcingaIcon($incidentIcon, ['class' => 'type-' . $this->item->type]);
+        if ($incidentIcon === '') {
+            $content = new IconBall('ellipsis', 'fa-solid');
```

Comment:
I understand your reasoning, but since the empty string is returned in only one case, please let this return an icon name instead and drop this handling here. And as icon, please use `notdef`.